src/Directory.Build.props

```diff
@@ -19,7 +18,7 @@
         <ApplicationIcon>../neo-cli.ico</ApplicationIcon>
     </PropertyGroup>
     <PropertyGroup>
-        <NeoVersion>3.6.2</NeoVersion>
+        <NeoVersion>3.7.5</NeoVersion>
```

Comment:
this can be removed now.

Reply:
You need to up projects to use `submodule` and stop using `nuget`

Reply:
It all in `neo` repo now

Reply:
fixed